### ISSUE TITLE
⛔ 🔗 Elementor Editor shows console error e.0 is not a function (Duplicate of #24492) 

### ISSUE THREAD

**Issue by rosskevin** [elementor/elementor#29111]
### Prerequisites

- [X] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [X] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

The previous issue was locked: https://github.com/elementor/elementor/issues/29092#issuecomment-2445023418

@nicholaszein 

I have updated `Elementor: 3.25.2` where `Elementor: 3.25.0`, but I still receive the same `Uncaught TypeError: e.O is not a function`.

Thoughts?

### Steps to reproduce

1. `Elementor: 3.25.2`
2. Edit a page that has a global widget

### Expected behavior

Javascript error `Uncaught TypeError: e.O is not a function`.

![SCR-20241029-miyx](https://github.com/user-attachments/assets/050058a8-d490-429e-9b0f-52dbf2825b96)

### Isolating the problem

- [X] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [X] This bug happens with the Hello Elementor theme active.
- [X] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

```txt
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.25.5
	MySQL version: mariadb.org binary distribution v10.11.7-MariaDB-1:10.11.7+maria~ubu2004
	PHP Version: 8.3.10
	PHP Memory Limit: 256M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.6.2
	Site URL: https://foo.com
	Home URL: https://foo.com
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 256M
	Max Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en_US
	Timezone: America/New_York
	Admin Email: spmacdonald@foo.com
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 3.0.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:131.0) Gecko/20100101 Firefox/131.0

== Active Plugins ==
	Companion Revision Manager
		Version: 1.6.2
		Author: Papin Schipper

	CookieYes | GDPR Cookie Consent
		Version: 3.2.7
		Author: CookieYes

	Disable Gutenberg
		Version: 3.2.1
		Author: Jeff Starr

	Doubly
		Version: 1.0.45
		Author: Unlimited Elements

	Elementor
		Version: 3.25.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.25.0
		Author: Elementor.com

	Enable Media Replace
		Version: 4.1.5
		Author: ShortPixel

	MalCare WordPress Security Plugin - Malware Scanner, Cleaner, Security Firewall
		Version: 5.81
		Author: MalCare Security

	Material Design Icons for Page Builders
		Version: 1.5.1
		Author: Photon WP

	Query Monitor
		Version: 3.16.4
		Author: John Blackbourn

	Safe SVG
		Version: 2.2.6
		Author: 10up

	Foo WP
		Version: 3.0.0
		Author: rosskevin

	Site Kit by Google
		Version: 1.138.0
		Author: Google

	Unlimited Elements for Elementor
		Version: 1.5.125
		Author: Unlimited Elements

	WP Consent API
		Version: 1.0.7
		Author: RogierLankhorst

	Yoast SEO
		Version: 23.7
		Author: Team Yoast

	Yoast SEO Premium
		Version: 23.7
		Author: Team Yoast


== Must-Use Plugins ==
	Kinsta Must-use Plugins
		Version: 3.1.1
		Author: Kinsta Team


== Elements Usage ==
	
	container : 10
		button : 9
		container : 91
		divider : 19
		form : 1
		heading : 84
		icon : 6
		image : 59
		spacer : 10
		template : 4
		testimonial : 3
		text-editor : 60
	footer : 1
		container : 9
		form : 1
		heading : 1
		icon-list : 3
		social-icons : 1
		theme-site-logo : 1
	header : 1
		button : 1
		container : 6
		mega-menu : 1
		theme-site-logo : 1
	page : 4
		container : 15
		heading : 5
		icon-list : 1
		image : 2
		text-editor : 4
	popup : 7
		blockquote : 1
		container : 13
		form : 2
		heading : 13
		html : 3
		image : 2
		spacer : 5
		text-editor : 7
	widget : 0
		blockquote : 1
		button : 2
		form : 1
		icon-list : 1
		image-box : 6
		spacer : 1
		text-editor : 4
	wp-post : 10
		blockquote : 13
		container : 16
		heading : 6
		image : 1
		text-editor : 26
	wp-page : 28
		blockquote : 5
		button : 17
		container : 483
		divider : 8
		form : 5
		gallery : 4
		heading : 235
		html : 9
		icon-box : 4
		icon-list : 6
		image : 63
		image-box : 16
		image-carousel : 6
		nested-accordion : 1
		nested-carousel : 1
		posts : 1
		spacer : 28
		template : 27
		testimonial : 14
		text-editor : 215
		ucaddon_arrow_navigation : 2
		ucaddon_blox_play_button : 7
		ucaddon_logo_marquee : 1
		ucaddon_remote_tabs : 1
		ucaddon_team_member_carousel : 2
		ucaddon_ue_google_maps : 1
		ucaddon_unlimited_timeline : 1
		video : 22
	single-post : 1
		container : 9
		heading : 3
		loop-carousel : 1
		post-info : 1
		posts : 1
		spacer : 1
		text-editor : 1
		theme-post-content : 1
		theme-post-excerpt : 1
		theme-post-featured-image : 1
		theme-post-title : 1
	loop-item : 2
		button : 1
		container : 2
		heading : 1
		post-info : 1
		theme-post-content : 1
		theme-post-title : 2
	error-404 : 1
		button : 1
		container : 1
		heading : 2
		theme-post-featured-image : 1


== Settings ==
	
	cpt_support: post, page
	disable_color_schemes: yes
	disable_typography_schemes: yes
	allow_tracking: yes
	unfiltered_files_upload: 1
	font_display: swap
	disabled_elements: nav-menu, progress-tracker, stripe-button, paypal-button, taxonomy-filter, lottie, code-highlight, facebook-page, facebook-embed, facebook-comments, facebook-button, breadcrumbs, sitemap, share-buttons, countdown, table-of-contents, reviews, testimonial-carousel, media-carousel, call-to-action, flip-box, price-table, price-list, hotspot, animated-headline, slides, login, portfolio, theme-page-title, theme-site-title, text-path, ucaddon_unlimited_grid, ucaddon_ue_unfold, ucaddon_content_slider_new, ucaddon_content_carousel_new, ucaddon_icon_carousel, ucaddon_image_card_carousel, wp-widget-block, wp-widget-custom_html, wp-widget-nav_menu, wp-widget-tag_cloud, wp-widget-rss, wp-widget-recent-comments, wp-widget-recent-posts, wp-widget-categories, wp-widget-text, wp-widget-search, wp-widget-meta, wp-widget-media_video, wp-widget-media_gallery, wp-widget-media_image, wp-widget-media_audio, wp-widget-archives, wp-widget-calendar, wp-widget-pages, read-more, sidebar, audio, counter, progress, inner-section, nested-tabs, rating, ucaddon_remote_counter_display, ucaddon_remote_control_bullet_navigation, theme-archive-title, archive-posts, loop-grid, link-in-bio, off-canvas, author-box, post-comments, post-navigation, video-playlist, link-in-bio-var-2, link-in-bio-var-3, link-in-bio-var-4, link-in-bio-var-5, link-in-bio-var-6, link-in-bio-var-7, search


== Features ==
	Custom Fonts: 1
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Active
	Additional Custom Breakpoints: Active by default
	Container: Active by default
	Upgrade Swiper Library: Active by default
	Nested Elements Performance: Active by default
	Optimized Control Loading: Active
	Optimized Markup: Active
	Conditionally load Swiper CSS files: Inactive by default
	Plugin Onboarding: Active by default
	CSS Smooth Scroll: Active by default
	Default to New Theme Builder: Active by default
	Hello Theme Header & Footer: Active by default
	Elementor Home Screen: Active by default
	Landing Pages: Active by default
	Nested Elements: Active
	Editor Top Bar: Active
	Pages Panel: Inactive by default
	Element Caching: Active
	Link In Bio: Active by default
	Floating Buttons: Active by default
	Elementor Editor Events: Inactive by default
	Atomic Widgets: Inactive by default
	Launchpad Checklist: Inactive by default
	Menu: Active


== Log ==
	
JS: showing 4 of 4JS: 2024-10-02 16:53:34 [error X 1][https://foo.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.24.2:3:54579] V is undefined 
JS: 2024-10-02 17:23:22 [error X 5][https://foo.com/wp-content/plugins/elementor/assets/js/frontend.min.js?ver=3.24.2:2:31655] elementor.settings.page is undefined 
JS: 2024-10-08 17:23:02 [error X 4][https://foo.com/wp-content/plugins/elementor/assets/js/common-modules.min.js?ver=3.24.5:2:56026] q.functionName is undefined 
JS: 2024-10-15 21:27:17 [error X 1][https://foo.com/wp-content/plugins/elementor/assets/js/video.78c625e89ab767d621c5.bundle.min.js:2:1642] this.youtubePlayer.playVideo is not a function 

Log: showing 20 of 352024-10-17 14:39:53 [info] elementor-pro::elementor_pro_updater Started 
2024-10-17 14:39:53 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-10-17 14:39:53 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-10-17 14:39:53 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.24.3',
  'to' => '3.24.4',
)]
2024-10-17 14:39:53 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.24.3',
  'to' => '3.24.4',
)]
2024-10-29 11:56:13 [info] elementor::elementor_updater Started 
2024-10-29 11:56:13 [info] Elementor/Upgrades - _on_each_version Start  
2024-10-29 11:56:13 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.24.7',
  'to' => '3.25.1',
)]
2024-10-29 11:56:15 [info] Elementor/Upgrades - _on_each_version Finished 
2024-10-29 11:56:15 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.24.7',
  'to' => '3.25.1',
)]
2024-10-29 11:57:02 [info] elementor-pro::elementor_pro_updater Started 
2024-10-29 11:57:02 [info] Elementor Pro/Upgrades - _on_each_version Start  
2024-10-29 11:57:02 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2024-10-29 11:57:02 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.24.4',
  'to' => '3.25.0',
)]
2024-10-29 11:57:02 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.24.4',
  'to' => '3.25.0',
)]
2024-10-29 14:43:35 [info] elementor::elementor_updater Started 
2024-10-29 14:43:35 [info] Elementor/Upgrades - _on_each_version Start  
2024-10-29 14:43:35 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.1',
  'to' => '3.25.2',
)]
2024-10-29 14:43:37 [info] Elementor/Upgrades - _on_each_version Finished 
2024-10-29 14:43:37 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.25.1',
  'to' => '3.25.2',
)]

PHP: showing 3 of 3PHP: 2024-10-04 13:36:04 [warning X 1][/www/fooweb50_197/public/wp-content/plugins/elementor/includes/elements/container.php::1874] Undefined array key "size" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2024-10-21 13:33:46 [notice X 37][/www/fooweb50_197/public/wp-content/plugins/elementor/includes/base/controls-stack.php::229] hexdec(): Passing null to parameter #1 ($hex_string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /www/fooweb50_197/public/wp-content/plugins/elementor/includes/base/controls-stack.php(229): class type hexdec()
#2: /www/fooweb50_197/public/wp-content/plugins/elementor/includes/base/controls-stack.php(243): Elementor\Controls_Stack -> get_id_int()
#3: /www/fooweb50_197/public/wp-content/plugins/elementor-pro/modules/mega-menu/widgets/mega-menu.php(2020): Elementor\Controls_Stack -> get_widget_number()
#4: /www/fooweb50_197/public/wp-content/plugins/elementor-pro/modules/mega-menu/widgets/mega-menu.php(2397): ElementorPro\Modules\MegaMenu\Widgets\Mega_Menu -> render_menu_wrapper_attributes()
',
)]
PHP: 2024-10-21 13:33:46 [notice X 37][/www/fooweb50_197/public/wp-content/plugins/elementor/includes/base/controls-stack.php::229] hexdec(): Passing null to parameter #1 ($hex_string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /www/fooweb50_197/public/wp-content/plugins/elementor/includes/base/controls-stack.php(229): class type hexdec()
#2: /www/fooweb50_197/public/wp-content/plugins/elementor/includes/base/controls-stack.php(243): Elementor\Controls_Stack -> get_id_int()
#3: /www/fooweb50_197/public/wp-content/plugins/elementor-pro/modules/mega-menu/widgets/mega-menu.php(2021): Elementor\Controls_Stack -> get_widget_number()
#4: /www/fooweb50_197/public/wp-content/plugins/elementor-pro/modules/mega-menu/widgets/mega-menu.php(2397): ElementorPro\Modules\MegaMenu\Widgets\Mega_Menu -> render_menu_wrapper_attributes()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified
	Unlimited Elements for Elementor: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
```

### Agreement

- [X] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hi @rosskevin, 

Closing and locking issues is standard procedure when they are resolved. If you still are experiencing issue after a patch is released, you can open a new bug report like you did here. 

However, the Console Error you're seeing is not related to the Global Widgets issue reported and solved in #29092.

There is already an issue about this subject here: #24492.

We are aware of this console error, and we know it's harmless. It's caused by the Site Settings' Style Guide. You can ignore it, it won't affect the functionality of Elementor.

Please, next time search before submitting your issue to avoid creating duplicates.

Kind regards

**Comment by nicholaszein** [comment]
Hello there!

We have awesome news! 🙌 🙌 🙌 

### 📢 We're happy to announce that this issue was FINALLY SOLVED in Elementor Core v3.27.0! 🥳 🎉 

The Style Guide no Longer Throws a Console Error **`e.0 is not a function`** anymore! 💪 

It took us a while to find the right spot to fix this issue while working on v4, but we finally found a spot!

Thank you for those who showed some patience! We really appreciate it! 🙏 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
